lib/Migration/Version001000Date20250720000000.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/** @noinspection PhpUnused */
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\Tables\Migration;
+
+use Closure;
+use OCP\DB\Exception;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version001000Date20250720000000 extends SimpleMigrationStep {
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 * @throws Exception
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		$table = $schema->getTable('tables_columns');
+		if (!$table->hasColumn('custom_settings')) {
+			$table->addColumn('custom_settings', Types::JSON, [
```

Comment:
There is no point in messing around with JSON here; plain TEXT will suffice.

Reply:
I'd prefer to be consistent with already existent columns https://github.com/nextcloud/tables/blob/main/lib/Migration/Version000600Date20230703000000.php#L68-L76